aurelia-templating-resources\index.d.ts(598,113): error TS1144: '{' or ';' expected

Hello, 
I'm submitting a bug report. 

Current behavior: 
When I build the project d.ts file causes the following error: 
`typings\modules\aurelia-templating-resources\index.d.ts(598,113): error TS1144: '{' or ';' expected.`

Corresponding line in the file is below: 
`export function createFullOverrideContext(repeat: any, data: any, index: any, length: any, key?: string): import("aurelia-binding").OverrideContext;`

Expected behavior:  
It should not generate a build error.  

Additional information: 
This line looks like it is recently changed. from `any` to `import("aurelia-binding").OverrideContext;`. 
'aurelia-binding' is actually imported on the top of the file. So changing the code to be as follows should solve the problem: 
`export function createFullOverrideContext(repeat: any, data: any, index: any, length: any, key?: string): OverrideContext;`

Library Version: 
1.5.4

Operating System:
Windows 10

Node Version:
6.9.1 

NPM Version:  
3.10.8 

JSPM Version: 
0.16.54 

Language: 
Typescript 2.4.2

--- a/src/repeat-utilities.ts
+++ b/src/repeat-utilities.ts
@@ -3,7 +3,8 @@ import {
   BindingBehavior,
   ValueConverter,
   sourceContext,
-  bindingMode
+  bindingMode,
+  OverrideContext
 } from 'aurelia-binding';
 
 const oneTime = bindingMode.oneTime;
@@ -31,7 +32,7 @@ export function updateOverrideContexts(views, startIndex) {
  * @param length The collections total length.
  * @param key The key in a key/value pair.
  */
-export function createFullOverrideContext(repeat, data, index, length, key?: string) {
+export function createFullOverrideContext(repeat, data, index, length, key?: string): OverrideContext {
   let bindingContext = {};
   let overrideContext = createOverrideContext(bindingContext, repeat.scope.overrideContext);
   // is key/value pair (Map)

